go/kex2/transport_test.go

```diff
@@ -462,7 +462,7 @@ func TestErrAgain(t *testing.T) {
 
 func TestPollLoopSuccess(t *testing.T) {
 
-	wait := time.Duration(50) * time.Millisecond
+	wait := time.Duration(100) * time.Millisecond
```

Comment:
i think that's the repoll interval, so i think it's unrelated, yes...

gee, i wish i could redo all of this stuff with a virtualized clock, but ... :(